runtime/compiler/ilgen/Walker.cpp

```diff
@@ -6719,6 +6719,31 @@ TR_J9ByteCodeIlGenerator::genAconst_init(TR_OpaqueClassBlock *valueTypeClass, in
    genFlush(0);
    }
 
+// returns true if call chain is StringBuilder.toString() -> String<init>(StringBuilder)
+//                -> String<init>(AbstractStringBuilder, Void) -> Arrays.copyOfRange([BII)
+// The inline depth must be at least 4 otherwise exception would occur.
+static bool isNewArrayResultOfStringBuilderToString(TR::Compilation *comp)
```

Comment:
Can we rename this bit more meaningful?

Reply:
Renamed to isCallChainFromStringBuilderDotToStringToArraysDotCopyOfRange